### DISCUSSION TITLE
Elastic Agent Installer Hangs Indefinitely

### DISCUSSION THREAD

**Discussion by teldredge** [Security-Onion-Solutions/securityonion#10080] [2.4]
I installed 2.4b (IMPORT) and opened the elastic-agent port in the firewall configuration.
I can telnet to 8220 on the SO server from a test client.
Got the agent installer powershell command and ran elevated on the test client.
Using the default agent policy 'Endpoints-Initial'
It downloads and extracts the archive but then hangs forever when running the elastic-agent.exe install command.
Also tried with the --insecure flag but same result.
Are there any local install logs I can look at to see what might be going on?

EDIT: I tried with --force and finally got an error message:
"Error: fail to enroll: fail to execute request to fleet-server: x509: certificate signed by unknown authority"
So I tried with both --force and --insecure flags and it looks like it registered!

**Comment by palichx** [comment]
Did you add client ip address in SOC-firewall-hostgroups-Elastic Agents

**Reply by teldredge** [reply]
Yes. Not sure why but adding the --insecure and --force flags to the install command seems to have done the trick. This is just a testing install so that's fine for now. Thanks.
